core/src/test/java/greencity/service/impl/TipsAndTricksCommentServiceImplTest.java

```diff
@@ -0,0 +1,221 @@
+package greencity.service.impl;
+
+import greencity.ModelUtils;
+import greencity.dto.PageableDto;
+import greencity.dto.tipsandtrickscomment.AddTipsAndTricksCommentDtoRequest;
+import greencity.dto.tipsandtrickscomment.AddTipsAndTricksCommentDtoResponse;
+import greencity.dto.tipsandtrickscomment.TipsAndTricksCommentDto;
+import greencity.entity.TipsAndTricksComment;
+import greencity.entity.User;
+import greencity.exception.exceptions.BadRequestException;
+import greencity.repository.TipsAndTricksCommentRepo;
+import greencity.service.TipsAndTricksService;
+import java.util.*;
+import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.modelmapper.ModelMapper;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+@ExtendWith(SpringExtension.class)
+public class TipsAndTricksCommentServiceImplTest {
+    @Mock
+    private TipsAndTricksCommentRepo tipsAndTricksCommentRepo;
+    @Mock
+    private TipsAndTricksService tipsAndTricksService;
+    @Mock
+    private ModelMapper modelMapper;
+    @InjectMocks
+    private TipsAndTricksCommentServiceImpl tipsAndTricksCommentService;
+
+    private TipsAndTricksComment tipsAndTricksComment = ModelUtils.getTipsAndTricksComment();
+    private AddTipsAndTricksCommentDtoResponse addTipsAndTricksCommentDtoResponse =
+        ModelUtils.getAddTipsAndTricksCommentDtoResponse();
+    private AddTipsAndTricksCommentDtoRequest addTipsAndTricksCommentDtoRequest =
+        ModelUtils.getAddTipsAndTricksCommentDtoRequest();
+    private TipsAndTricksCommentDto tipsAndTricksCommentDto = TipsAndTricksCommentDto.builder().id(1L).build();
```

Comment:
Modifying and using the same object in different tests influences test execution and can have unpredictable result.